### ISSUE TITLE
Light Gnome Shell Theme

### ISSUE THREAD

**Issue by bremaya** [ubuntu/yaru#3903]
Because doens't have an full white theme in Ubuntu 22.04?

**Comment by Feichtmeier** [comment]
I agree 😄 We support light and dark theme but the light theme uses a dark panel. I guess this is what you meant?

There is actually no reason to make the panel not lighter for the light theme. It doesn't necessarily need to be full white but something brighter would be better.

Since yaru is derived from the pure upstream gnome shell theme, we hoped for some time that upstream will also support a light theme and we do not need to modify the source to support both themes every single gnome shell release. 

@Muqtxdir you are the upstream expert ;) do you know the current progress on the light theme upstream?

**Comment by Jupi007** [comment]
I started to work on a POC on my computer last month. Honestly, it looks better to have a full light theme.

But yeah, @Muqtxdir I'm curious to know if upstream is working on something similar.

**Comment by Muqtxdir** [comment]
Hi everyone,

Apologies for the delayed response, With GNOME 45, the `gnome-shell` build now generates both `gnome-shell-dark` and `gnome-shell-light`. Although the way to toggle to the light-theme would requuire the user-themes extension

**Comment by Jupi007** [comment]
Thanks @Muqtxdir 
I started to work on the Yaru sync a few months ago, but I was too busy.
Btw does upstream finalized the full light theme, or is it still partial?

**Comment by Muqtxdir** [comment]
One approach that I can suggest for this issue:
- `gnome-shell` (default)
- `gnome-shell-light` (prefer-light)
- -gnome-shell-dark` (prefer-dark)

and we can have gnome-shell switch/swap stylesheets based on the patch from @3v1n0 that allows `color-scheme` profile from `org.gnome.shell.ubuntu`

**Comment by Muqtxdir** [comment]
> Thanks @Muqtxdir
I started to work on the Yaru sync a few months ago, but I was too busy.
Btw does upstream finalized the full light theme, or is it still partial?


@Jupi007 its full light-theme in upstream not like ours

**Comment by Jupi007** [comment]
> @Jupi007 its full light-theme in upstream not like ours

Oh great, the last time only the panel and menus was light!
I'll checkout that :)

**Comment by julienfalque** [comment]
Gnome 45 with default theme supports having a light shell with the [Light Style extension](https://extensions.gnome.org/extension/6198/light-style/), is that what this discussion is about?
